public/geometries/franka_description/README.md

```diff
@@ -0,0 +1,3 @@
+This is modified version of https://github.com/frankaemika/franka_ros/tree/develop/franka_description 
+that includes a `.urdf` file instead of a `.xacro` file that everyone would need to compile. It also replaces the normal hand with the `robotiq_arg85` hand.
```

Comment:
`rm public/geometries/franka_description/robots/common/*.xacro` if not needed

Reply:
removed all `*.xacro` files